.github/workflows/code-coverage-test.yml

```diff
@@ -0,0 +1,624 @@
+name: Code Coverage Test
+
+env:
+  # TODO: we really need to define a list of supported versions (ideally it's no more than 2)
+  #       build is done on the lowest version and test on the highest with a "sanity test"
+  #       stage done on all versions in the list ecept the highest
+  EL8_BUILD_VERSION: 8.6
+  EL8_VERSION: 8.8
+  EL9_BUILD_VERSION: 9
+  EL9_VERSION: 9
+  LEAP15_VERSION: 15.5
+
+  JENKINS_URL: https://build.hpdd.intel.com/
+  # DAOS_REPOSITRY: master
+  DAOS_REPOSITRY: pahender/DAOS-16285_master
+
+on:
+  pull_request:
+  schedule:
+    - cron: "0 0 * * 6"   # 'TZ=UTC\n0 0 * * 6'
+
+concurrency:
+  group: rpm-build-and-test-${{ github.head_ref  || github.run_id }}
```

Comment:
I think you need to change the prefix here.  Technically (probably but not entirely probably) this could collide with https://github.com/daos-stack/daos/blob/85959c5899e4f96ebd83bb32dd685fda87f85dd7/.github/workflows/rpm-build-and-test.yml#L22

Basically what's before the `-${{ github.head_ref  || github.run_id }}` should be unique per workflow.